using-valtimo/plugin/portaaltaak/configure-portaaltaak-plugin.md

```diff
@@ -0,0 +1,118 @@
+definition to handle the completion itself. This can be done using the [Complete Portaal taak](#complete-portaal-taak) 
+plugin action. The process 'Process Portaaltaak uploaded Documents' that is shipped with Valtimo can be used here. See 
+[this section](#configuring-the-task-completion-process ) on how to set up this process. 
+
+An example plugin configuration:
+![example plugin configuration](img/configure-plugin.png)
+
+## Configuring the task completion process 
+
+When a task object is updated and Valtimo gets the notification, a process is started to handle additional steps needed 
+for completing the task. Because the task can only be considered completed when all other processing of data has 
+happened, the actual completion of the task is also a plugin action that can be configured in this process.
+
+The process that is started in configured in the plugin properties by setting the 'Uploaded documents handler process'
+property. Valtimo ships with the `Process Portaaltaak uploaded Documents` process which has two user tasks.
+
+![Process Portaaltaak uploaded Documents](img/process-portaal-taak.png)
+
+These user tasks need to be configured with process links before the process can be used. The following actions should 
+be configured:
+- Link document to zaak - [Link document to zaak](../zaken-api/configure-zaken-api-plugin.md#link-document-to-zaak) in the Zaken API plugin
+- Update Portaal Taak Status - [Complete Portaal taak](#complete-portaal-taak) in the Portaal Taak plugin
+
+Instead of using the `Process Portaaltaak uploaded Documents` process it is possible to create another process that will
+handle task completion in a different way. The process is started with a few process variables that can be used when 
+designing another process definition. These variables are:
+- **portaalTaakObjectUrl.** The URL of the taak object.
+- **objectenApiPluginConfigurationId.** The id of the Objecten API plugin configuration that can be used to get or edit 
+the taak object.
+- **verwerkerTaakId.** The id of the of the task to be completed. 
+- **documentUrls.** A list of document URLs of documents stored in the Documenten API. Can be used as Collection in BPMN
+multi-instance elements to iteratte over the list.
+
+![img.png](img/document-urls-collection-example.png)
+
+## Available actions
+
+The portaal taak plugin supports the following actions that can be configured in process links in order to create and 
+complete user tasks through external systems like the NL Portal.
+
+A general description on how to create process links, can be found [here](../create-process-link.md).
+
+### Create Portaal taak
+        
+The **Create portal task** action is linked to a user task that can be completed in an external system. A taak object
+is created when the task is reached. This taak object is created in the Objecten API configured for the object 
```

Comment:
You're almost saying the same thing twice here. Maybe rephrase to something like:

`A taak object is created when the task is reached, in the Objecten API configured for the object management configuration for the plugin configuration.`

Not sure what the last half of this sentence should be by the way. I just copied what you stated, but I don't understand what you're trying to say.